recipes/sdl/all/conanfile.py

```diff
@@ -130,7 +130,6 @@ def requirements(self):
                 self.requires("opengl/system")
             if self.options.nas:
                 self.requires("nas/1.9.4")
-                self.requires("xorg/system")
```

Comment:
Not sure why it was added when `nas=True`, but shouldn't it be at least added when `wayland=False`?

Reply:
I've tried and it worked without that requirement. I would have expected it to fail, but... maybe we have too many things installed in our docker images?